src/native/minipal/cpufeatures.c

```diff
@@ -277,6 +277,28 @@ int minipal_getcpufeatures(void)
                                             {
                                                 result |= XArchIntrinsicConstants_AvxVnni;
                                             }
+
+                                            if ((cpuidInfo[CPUID_EDX] & (1 << 19)) != 0)                                // Avx10
+                                            {
+                                                __cpuidex(cpuidInfo, 0x00000024, 0x00000000);
+                                                const int versionMask = 0xFF; // [7:0]
+                                                if((cpuidInfo[CPUID_EBX] & versionMask) >= 1) // version higher than 1
+                                                {
+                                                    result |= XArchIntrinsicConstants_Avx10v1;
```

Comment:
nit: We should explicitly check `EBX[16]` is set to indicate `AVX10/V128`. The bit exists, so we should be explicit about checking it.